wp-content/themes/pub/wporg-learn-2024/src/style/_rtl.scss

```diff
@@ -0,0 +1,44 @@
+/**
+ * Styles for RTL sites.
+ * Note, these styles are set here using the LTR layout keywords, so that when
+ * `rtlcss` runs it flips the right/left to the correct values.
+ */
+
+body.rtl {
+
+	// Override the inline styles on Learning Pathway Card content.
+	.wporg-learn-learning-pathway-card-header-content {
+		padding-right: 0 !important;
+		padding-left: 30px !important;
+	}
+
+	// Override the alignment on Sensei buttons.
+	.wp-block-sensei-lms-button-take-course.has-text-align-left {
+		text-align: unset;
+	}
+
+	// Flip the chevrons on Sensei lesson listings, "back to lesson".
+	.wp-block-sensei-lms-course-outline-lesson__chevron,
+	.sensei-lms-quiz-back-to-lesson svg {
+		transform: rotate(180deg);
+	}
+
+	// Override the space between quiz inputs and question labels.
+	&.quiz form #sensei-quiz-list li ul li input {
+		margin: 0 10px 0 0;
+	}
+
+	// Fix the border on the Sensei lesson page.
```

Comment:
This border isn't actually supposed to be displayed, it's a Sensei thing and the design doesn't show it. An issue was opened and closed to be addressed in https://github.com/WordPress/Learn/issues/2638. So I think we can remove this and unset the border and padding for all modes.

Reply:
Removed it in https://github.com/WordPress/Learn/pull/2669/commits/62adb0cf2817e868147bd0be0e9feb24975478dc